DashKitUI/View Controllers/DashUICoordinator.swift

```diff
@@ -183,13 +192,10 @@ class DashUICoordinator: UINavigationController, PumpManagerCreateNotifying, Pum
             if let pumpManager = pumpManager {
                 let vc = PodSetupCompleteViewController.instantiateFromStoryboard(pumpManager, navigator: self)
                 vc.completion = { [weak self] in
-                    if let self = self {
-                        let settings = PumpManagerSetupSettings(maxBasalRateUnitsPerHour: self.maxBasalRateUnitsPerHour,
-                                                           maxBolusUnits: self.maxBolusUnits,
-                                                           basalSchedule: self.basalSchedule)
-                        self.pumpManagerOnboardDelegate?.pumpManagerOnboardNotifying(didOnboardPumpManager: pumpManager, withFinalSettings: settings)
-                        self.stepFinished()
+                    if let self = self, let initialSettings = self.initialSettings {
+                        self.pumpManagerOnboardDelegate?.pumpManagerOnboardNotifying(didOnboardPumpManager: pumpManager, withFinalSettings: initialSettings)
```

Comment:
Will the pod setup ever change the max basal, max bolus, or basal schedule? If so, do we need to return those updated values rather than the initial settings passed in? Or are the initial settings the only settings now?

Reply:
For pods, we will not change the settings at all. I think it's worth leaving the current possibility open, though I don't know that any pump managers will use it currently.